ydb/core/tx/schemeshard/ut_helpers/export_reboots_common.cpp

```diff
@@ -11,20 +11,36 @@ using namespace NKikimrSchemeOp;
 namespace NSchemeShardUT_Private {
 namespace NExportReboots {
 
+using TCreateHandler = std::function<void(TTestActorRuntime&, ui64&, const TString&, const TString&)>;
+
+#define GEN_CREATE_HANDLER(func) \
+    TCreateHandler([](TTestActorRuntime& runtime, ui64& txId, const TString& parentPath, const TString& scheme) { \
+        func(runtime, txId, parentPath, scheme); \
+    })
+
+THashMap<NKikimrSchemeOp::EPathType, TCreateHandler> CreateHandlers = {
+    {EPathTypeTable, GEN_CREATE_HANDLER(TestCreateTable)},
+    {EPathTypeView, GEN_CREATE_HANDLER(TestCreateView)},
+    {EPathTypeCdcStream, GEN_CREATE_HANDLER(TestCreateCdcStream)},
+};
+
+void TestCreate(
+    TTestActorRuntime& runtime,
+    ui64& txId,
+    const TString& scheme,
+    NKikimrSchemeOp::EPathType pathType
+) {
```

Comment:
Вполне влезает в одну строку и `txId` не нужно передавать по ссылке:
```suggestion
void TestCreate(TTestActorRuntime& runtime, ui64 txId, const TString& scheme, NKikimrSchemeOp::EPathType pathType) {
```